paymentsheet/src/main/java/com/stripe/android/paymentsheet/navigation/PaymentSheetScreen.kt

```diff
@@ -223,7 +223,7 @@ internal sealed interface PaymentSheetScreen {
 
         @Composable
         override fun Content(modifier: Modifier) {
-            AddPaymentMethod(interactor = interactor, modifier)
+            AddPaymentMethod(interactor = interactor)
```

Comment:
We weren't passing the modifier for the `VerticalModeFormUI`. Looks like because the form applies its own padding. Should we move the default modifier that applies `8.dp` as bottom padding into the individual screens.

Reply:
I don't think the individual screens should be responsible for their own padding. In this case, we would need to add 8.dp to all the screens individually? So it just makes more sense to apply that padding globally.

Rather than doing this, I think we should remove the padding from the forms, if we need to